users/serializers.py

```diff
@@ -210,6 +212,7 @@ def update(self, instance, validated_data):
 
             user = super().update(instance, validated_data)
 
+        sync_hubspot_user(user)
```

Comment:
So I was testing this thing and I noticed that with every update in the user through `/profile` the updated name on Hubspot was being set to what it was before the update. e.g. you can try running the flow like this:

1. Go to `/profile`
2. Change your First or Last name and click update
3. Notice the name on Hubspot

In my case the name there is always a step behind which could be because the overridden update method returns a lazy user object and i think it's possible that the user hasn't been really updated in the DB yet while we sync it with Hubspot that's why it's always one step behind.

Should we move this into a signal maybe but that depends on how your test turns out if you follow the same steps above.

Reply:
I wasn't able to duplicate this, any change I made to name on the profile page was also updated in Hubspot, but sometimes it took a few minutes for the change to appear.  Can you try again but give Hubspot up to ~5 minutes or so to reflect the change?

Reply:
Thanks for the details, This worked.